src/pybamm/parameters/parameter_sets.py

```diff
@@ -18,7 +18,7 @@ class ParameterSets(Mapping):
 
         >>> import pybamm
         >>> list(pybamm.parameter_sets)
-        ['Ai2020', 'Chen2020', ...]
+        ['Sulzer2019', 'Ai2020', 'Chen2020', 'Chen2020_composite', 'Ecker2015', 'Ecker2015_graphite_halfcell', 'Marquis2019', 'Mohtat2020', 'NCA_Kim2011', 'OKane2022', 'OKane2022_graphite_SiOx_halfcell', 'ORegan2022', 'Prada2013', 'Ramadass2004', 'Xu2019', 'ECM_Example', 'MSMR_Example']
```

Comment:
The new test runner using `pytest` seems to need the complete output. Otherwise, it was failing the tests.

Reply:
I don't recall seeing failures in other PRs due to this. Is the failure something scikit-build-core specific?

Reply:
I will try reverting the change. If it works with just an example, then that would be great. Otherwise, it might be related to scikit-build-core (I don't know how this is even related, to be honest).

Reply:
Yes, I had the same question, but I haven't figured out the problem. My hunch is that it changes the order of the files listed in the wheel metadata in comparison to the order set up in the `setuptools` build, causing `import pybamm` to change the order of what files are picked up first (and therefore this is seen in the doctests). Of course, this is only a hunch...